rca/settings/base.py

```diff
@@ -722,10 +722,18 @@
 # Birdbath
 BIRDBATH_CHECKS = [
     "birdbath.checks.contrib.heroku.HerokuNotProductionCheck",
-    "birdbath.checks.contrib.heroku.HerokuAnonymisationAllowedCheck",
+]
+BIRDBATH_PROCESSORS = [
+    "birdbath.processors.users.UserEmailAnonymiser",
+    "birdbath.processors.users.UserPasswordAnonymiser",
+    "birdbath.processors.contrib.wagtail.SearchQueryCleaner",
+    "birdbath.processors.contrib.wagtail.FormSubmissionCleaner",
+    "rca.enquire_to_study.birdbath.EnquiryFormSubmissionDeleter",
+    "rca.scholarships.birdbath.ScholarshipEnquiryFormSubmissionDeleter",
 ]
 BIRDBATH_REQUIRED = env.get("BIRDBATH_REQUIRED", "true").lower() == "true"
-BIRDBATH_USER_ANONYMISER_EXCLUDE_EMAIL_RE = r"torchbox\.com$"
+BIRDBATH_USER_ANONYMISER_EXCLUDE_SUPERUSERS = True
+BIRDBATH_USER_ANONYMISER_EXCLUDE_EMAIL_RE = r"(torchbox\.com|rca\.ac\.uk)$"
```

Comment:
It looks like students have emails with `@network.rca.ac.uk`, and a small number has `@rca.ac.uk`. We will want to anonymise their accounts. 

Editors seem to have `@rca.ac.uk` emails, but there is no easy way to differentiate them from students using the excluder in the user anonymiser from birdbath.

So unless they specifically want to retain the accounts of moderators and editors who are not superusers, you could revert the `BIRDBATH_USER_ANONYMISER_EXCLUDE_EMAIL_RE` back to BIRDBATH_USER_ANONYMISER_EXCLUDE_EMAIL_RE = r"torchbox\.com$". Otherwise you could possibly customise the anoymiser to take into account user groups.

Reply:
I have made changes here, see blow.